### ISSUE TITLE
how to identify anomaly in LSTM prediction?

### ISSUE THREAD

**Issue by KiteFlyKid** [morrigan/user-behavior-anomaly-detector#2]
Is the LSTM  a regression model to predict anomaly?
If so, after you have calculated the rmse score of the coming action in the LSTM forecast function,  how do you identify an anomaly based on that score?

**Comment by vinal-2** [comment]
Yes, the LSTM in the project is a regression model to predict anomalies. It is used to forecast the next action that a user will take. If the predicted action is significantly different from the actual action, then this may indicate an anomaly.

They use the RMSE score to identify anomalies set at 0.5, so any predicted value that is more than 0.5 away from the actual value is considered an anomaly.
